### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Marketionist**
**This pull request adds more installation and function descriptions to README.md**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,34 @@
...
 
-```randomDigits(digitsNumber)``` - utility to generate a number of random digits
+## Installation
+Install `cms-protractor-helper` via [npm](https://www.npmjs.com/package/cms-protractor-helper "cms-protractor-helper on npmjs.com"):
 
-```waitFor(element)``` - utility to wait for element to appear on the page
+    npm install cms-protractor-helper --save-dev
 
-```waitForDisplayed(element)``` - utility to wait for element to be displayed on the page
+Or just add it to package.json:
```
Comment (vxsx) [inline README.md]: we should never never advise this

Comment (FinalAngel) [inline README.md]: agreed

File: README.md [hunk 2]
```diff
@@ -1,15 +1,34 @@
...
+
+`cmsProtractorHelper.waitFor(element);` - utility to wait for element to appear on the page
+
+`cmsProtractorHelper.waitForDisplayed(element);` - utility to wait for element to be displayed on the page
+
+`cmsProtractorHelper.expectToBeAbsent(element);` - utility to test the expectation that the element is missing
+
+`cmsProtractorHelper.selectOption(select, optionText, option);` - utility to select the option
+
+`cmsProtractorHelper.login({ data });` - utility to log in to the site: `{ data }` object can contain credentials: site, username and password, if not set it will fallback to 'http://127.0.0.1:8000/', 'admin', 'admin'. Also `{ data }` object should contain elements selectors needed for the login process - for example:
```
Comment (vxsx) [inline README.md]: no need to wrap `data` in brackets, it's ok to say that this is an object.
`{ data }` is actually valid code in es6 and depending on position means either destructuring assignment or shorthand object creation

File: README.md [hunk 3]
```diff
@@ -1,15 +1,34 @@
...
 ```
 // login to the site
 cmsProtractorHelper.login(
```
Comment (vxsx) [inline README.md]: should put the curly bracket which is currently on next line to this line

Comment (Marketionist) [inline README.md]: Fixed

File: README.md (hunk 2 above)
Comment (Marketionist) [inline README.md]: Fixed

File: README.md (hunk 1 above)
Comment (Marketionist) [inline README.md]: Removed

**Comment by vxsx** [comment]
LGTM

**Pull request merged**